pints/tests/test_mcmc_hamiltonian.py

```diff
@@ -138,6 +138,19 @@ def test_set_hyper_parameters(self):
         self.assertEqual(mcmc.leapfrog_step_size()[0], 1.5)
         self.assertEqual(mcmc.leapfrog_step_size()[1], 3)
 
+    def test_other_setters(self):
+        """
```

Comment:
For a stupid technical reason these shouldn't be docstrings but just inline comments ```# These guys```

Reply:
Done